bus-mapping/src/operation.rs

```diff
@@ -652,6 +652,12 @@ pub enum CallContextField {
     IsPersistent,
     /// IsStatic
     IsStatic,
+    /// LastCalleeId
+    LastCalleeId,
```

Comment:
just thinking how about if we only record `LastCalleeId` without `LastCalleeReturnDataOffset` `LastCalleeReturnDataOffset` ? then we can  look up those two items by `LastCalleeId` as already have `ReturnDataOffset` `ReturnDataOffset` , something missing ?

Reply:
Fields `LastCalleeReturnData*` are decided by the callee's step which halts.

Fields `ReturnData*` are decided by the caller's step which triggers the call, to ask callee to copy the `ReturnData` to the specified range.

So they are for different purposes.